it/content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -8,4 +8,5 @@
     <filter root="/conf/core-components-it" mode="update"/>
     <filter root="/conf/wknd" mode="update"/>
     <filter root="/conf/datalayerContainer" mode="update"/>
+    <filter root="/content/core-components-examples/aemembedcontainertest"/>
```

Comment:
All sites specific use-case should be present under `/content/forms/sites`. No changes in existing filter is allowed unless there are new use-cases